chromadb/test/test_api.py

```diff
@@ -1100,3 +1100,21 @@ def test_get_include(api_fixture, request):
     with pytest.raises(ValueError) as e:
         items = collection.get(include=None)
     assert "Include" in str(e.value)
+
+
+# make sure query results are returned in the right order
+@pytest.mark.parametrize("api_fixture", test_apis)
+def test_query_order(api_fixture, request):
+    api = request.getfixturevalue(api_fixture.__name__)
+
+    api.reset()
+    collection = api.create_collection("test_query_order")
+    collection.add(**records)
+
+    items = collection.query(
+        query_embeddings=[1.2, 2.24, 3.2],
+        include=["metadatas", "documents", "distances"],
+        n_results=2,
+    )
+
+    assert items["documents"][0][0] == "this document is second"
```

Comment:
Should we add
assert items["documents"][0][1] == "this document is first"

Reply:
yea - good idea. ill add that